.github/workflows/node.yaml

```diff
@@ -0,0 +1,38 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04, macos-latest, windows-latest]
+        node-version: [12.x, 13.x]
+        redis-version: [4, 5]
+
+    steps:
+      - name: Git Checkout
+      - uses: actions/checkout@v2
+
+      - name: Start Redis
+      - uses: superchargejs/redis-github-action@1.0.0
+        with:
+          redis-version: ${{ matrix.redis-version }}
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - run: [
+            npm install
+            cp env.example .env
```

Comment:
Our tests don't currently do this anywhere, so we'll have to watch how this goes.